src/bugs/create.ts

```diff
@@ -4,7 +4,7 @@ import plugins from '../plugins';
 
 type Bug = {
     bid: number,
-    title: string,
+    name: string,
     description: string,
     timestamp: string,
```

Comment:
Do we still need this timestamp? I think this was removed from the BugObject.yaml.

Reply:
Removed